src/host/settings.cpp

```diff
@@ -283,7 +283,7 @@ void Settings::Validate()
     {
         // TODO: FIX
         //// Get the font that we're going to use to convert pixels to characters.
-        //DWORD const dwFontIndexWant = FindCreateFont(_uFontFamily,
+        // const auto dwFontIndexWant = FindCreateFont(_uFontFamily,

```

Comment:
>const auto dwFontIndexWant [](http://example.com/codeflow?start=11&length=26)

huh.

Reply:
ah no, I was merely observing that this refactor also applied to _commented out code_. There's a few other places too. Kinda neat IMO.

Reply:
I had to touch up a couple hundred lines of code because clang-format / Resharper didn't get it. In some places for instance `auto const` wasn't replaced with `const auto`.
I've used `\b(\w+) const (\w+) =` to find code like that and fixed that code by hand, including this comment.